Sources/SwiftGen/Config/Config.swift

```diff
@@ -121,26 +130,25 @@ extension Config {
       logger(.error, "output_dir: Output directory \(Message.doesntExist(outputDir))")
     }
 
-    for (cmd, entries) in commands {
-      if let replacement = Config.deprecatedCommands[cmd] {
-        logger(.warning, Message.deprecatedAction(cmd, for: replacement))
+    let groupedCommands: [ParserCLI: [ConfigEntry]] = Dictionary(grouping: commands) { $0.command }
+      .mapValues { $0.map { $0.entry } }
```

Comment:
Is the re-grouping only used so that we log "N entries for command X"? (Which we should s/command/parser/ btw)? If so, I'm not sure it's worth it tbh; we can just change the logs to print one line per config entry even if there are multiple entries using the same parser and get rid of the "N entries" count in log message if that's the only reason for Dictionary(grouping:…) to be there.

Reply:
Ah wait that's just for when running `config lint` (I thought this was for `config run --verbose`, missed the context in Mobile GitHub and focused snippets review) this is fine then.